packages/actor-init-sparql/lib/ActorInitSparql-browser.ts

```diff
@@ -169,6 +169,32 @@ export class ActorInitSparql extends ActorInit implements IActorInitSparqlArgs {
     // Execute query
     const resolve: IActionQueryOperation = { context, operation };
     const output = await this.mediatorQueryOperation.mediate(resolve);
+    // Set bindings
+    if ((<any> output).bindingsStream) {
+      (<IActorQueryOperationOutputBindings> output).bindings = () => {
+        const result: any[] = [];
+        (<IActorQueryOperationOutputBindings> output).bindingsStream.on('data', data => {
+          result.push(data.toObject());
+        });
+        return new Promise((resolveP, reject) => {
```

Comment:
```suggestion
        return new Promise((resolve, reject) => {
```

Reply:
Also, reject is never called. You should also listen to error events and forward those.